core/src/main/java/edu/wpi/grip/core/Pipeline.java

```diff
@@ -126,7 +127,7 @@ public void clear() {
    * @param <R>              The return type of the function
    * @return The value returned by the function.
    */
-  private final <R> R readStepsSafely(Function<List<Step>, R> stepListFunction) {
+  private <R> R readStepsSafely(Function<List<Step>, R> stepListFunction) {
```

Comment:
Is this for Guice?

Reply:
IIRC it was PMD or errorprone flagging an unneeded `final` modifier on a private method. Guice doesn't come into it